test/Microsoft.AspNetCore.Mvc.ViewFeatures.Test/Internal/PagedBufferedStringWriterTest.cs

```diff
@@ -44,6 +44,24 @@ static PagedBufferedStringWriterTest()
         }
 
         [Fact]
+        public async Task Write_CharArray_Null()
+        {
+            // Arrange
+            var pool = new TestArrayPool();
+            var inner = new StringWriter();
+
+            var writer = new PagedBufferedTextWriter(pool, inner);
+
+            // Act
+            writer.Write((char[])null);
+
+            await writer.FlushAsync();
+
+            // Assert
+            Assert.True(string.IsNullOrEmpty(inner.ToString()));
```

Comment:
Stop and think before you use `Assert.True` or `Assert.False` - it's almost always wrong.

Why is it null or empty?